scripts/model_quality_gate.py

```
@@ -0,0 +1,300 @@
+                "Baseline tag mismatch between --baseline-tag and --baseline-metrics-json: "
+                f"{args.baseline_tag!r} != {file_baseline_tag!r}"
+            )
+
+        file_fixture = baseline_metrics_file.get("fixture")
+        if file_fixture:
+            expected_fixture = str(args.fixture)
+            if file_fixture != expected_fixture:
+                raise ValueError(
+                    "Fixture mismatch between --fixture and --baseline-metrics-json: "
+                    f"{expected_fixture!r} != {file_fixture!r}"
+                )
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)